src/AccessibilityInsights.SharedUx/Properties/Resources.resx

```diff
@@ -1524,4 +1528,64 @@ Do you want to change the channel? </value>
   <data name="AutomatedChecksControl_RuleSourceLabel" xml:space="preserve">
     <value>Rule source</value>
   </data>
+  <data name="ApplicationSettingsControl_Fontsize" xml:space="preserve">
+    <value>Font Size</value>
+  </data>
+  <data name="ApplicationSettingsControl_milliseconds" xml:space="preserve">
+    <value>milliseconds</value>
+  </data>
+  <data name="DisplayCount" xml:space="preserve">
+    <value>5</value>
+  </data>
+  <data name="HierarchyControl_Control" xml:space="preserve">
+    <value>_Control</value>
+  </data>
+  <data name="HierarchyControl_Content" xml:space="preserve">
+    <value>Con_tent</value>
+  </data>
+  <data name="HierarchyControl_Raw" xml:space="preserve">
+    <value>_RawForReals</value>
```

Comment:
"ForReals" looks like a typo

Reply:
Yeah ... not sure how that got in there ... 🤦‍♂️